### PR TITLE
LPS-107030 SF - Move logic into *Util class

### PR DESCRIPTION
**Pull request by adolfopa**
Hey @ambrinchaudhary,

I think we should change the "siteAvailableLocales" and "siteDefaultLocale" to something more generic (e.g. "groupAvailableLocales"). We'll be using this for both sites and depots, so the attr. name should not refer to sites specifically.

Other than that, it looks good 👍 
Thanks!

### CODE REVIEW

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:test:stable

**Comment by ambrinchaudhary** [comment]
Just started reviewing :)

File: modules/apps/depot/depot-web/src/main/resources/META-INF/resources/js/Languages.es.js [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+ */
+const Languages = ({
+	availableLocales,
+	defaultLocaleId,
+	inheritLocales = false,
+	siteAvailableLocales,
+	siteDefaultLocaleId
+}) => {
+	const defaultOption = 'defaultOption';
+	const customOption = 'customOption';
```
Comment (boton) [inline modules/apps/depot/depot-web/src/main/resources/META-INF/resources/js/Languages.es.js]: Move outside and tye in UPPER_CASE

**Comment by ambrinchaudhary** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/83386. [See changes here.](https://github.com/ambrinchaudhary/liferay-portal/compare/658c961256ef~2...658c961256ef)

**Pull request closed without merging**